case-server/src/main/java/com/powsybl/caseserver/CaseService.java

```diff
@@ -39,6 +44,13 @@
 
     private ComputationManager computationManager = LocalComputationManager.getDefault();
 
+    private EmitterProcessor<Message<String>> caseInfosPublisher = EmitterProcessor.create();
```

Comment:
We need both a subscriber and a publisher (Flux<INT, OUT>) in order to emit message **manually** via the subscriber and publish it automatically (publisher) to the case.import topic.
The generate method is a programmatic creation that automatically generates a flow.

Reply:
We don't *need* a subscriber (onNext()) to generate items, it can be done like this as well:
```java
AtomicReference<FluxSink<String>> s = new AtomicReference<>();
ConnectableFlux<String> f = Flux.<String>create(sink -> {
s.set(sink);
}).publish();
f.connect();
FluxSink<String> sink = s.get();

//now use the sink
sink.next("foo");
```

like what they say in the docs 
> providing a “sink” that is similar in concept to a Processor, in the sense that it lets you manually populate the sequence with data or terminate it)

Not sure about the differences and which one is best for us. What do you think ?

Reply:
Some person on the internet said: "Also, consider using Processors, storing sink as a field is discouraged." (https://stackoverflow.com/a/56063933 )